### PR TITLE
Add package.json and ts

### PR DESCRIPTION
**Pull request by mfucci**
Add package.json and minimal ts shell so this can be published to npm

### CODE REVIEW

**Review by turon — APPROVED** [review]
LGTM :+1:

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "matter.js",
+  "version": "0.0.0",
```
Comment (turon) [inline package.json:3]: Perhaps 0.0.1?

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "CommonJS",
+    "outDir": "build",
```
Comment (turon) [inline tsconfig.json:5]: I thought `dist` is more common?

File: package.json (hunk 1 above)
Comment (mfucci) [reply package.json:3]: Discussed offline, 0.0.0 is to remove any expectation on npm that this contains something functional

File: tsconfig.json (hunk 2 above)
Comment (mfucci) [reply tsconfig.json:5]: Correct. Updated to dist

**Pull request merged**